pyatlan/client/atlan.py

```diff
@@ -262,26 +262,45 @@ def _call_api_internal(self, api, path, params, binary_data=None):
     def _call_api(
         self, api, query_params=None, request_obj=None, exclude_unset: bool = True
     ):
-        params, path = self._create_params(
+        path = self._create_path(api)
+        params, request_id = self._create_params(
             api, query_params, request_obj, exclude_unset
         )
+        if LOGGER.isEnabledFor(logging.DEBUG):
+            LOGGER.debug("------------------------------------------------------")
+            LOGGER.debug("Call         : %s %s", api.method, path)
+            LOGGER.debug("Content-type_ : %s", api.consumes)
+            LOGGER.debug("Accept       : %s", api.produces)
+            LOGGER.debug("Request ID   : %s", request_id)
         return self._call_api_internal(api, path, params)
 
+    def _create_path(self, api: API):
+        if self.base_url == "INTERNAL":
+            return os.path.join(api.endpoint.service, api.path)
```

Comment:
Does this use an OS-specific path separator when joining the components together? (If so, won't this use `\` on Windows, and thus not be a valid URL?)

Reply:
Good catch. The use of os.path.join was inherited from the apache atlan API.  I have changed it to use urljoin.